kale/prepdata/tabular_transform.py

```diff
@@ -0,0 +1,87 @@
+# =============================================================================
+# Author: Sina Tabakhi, sina.tabakhi@gmail.com
+# =============================================================================
+
+"""
+Provide data transformations for tabular datasets.
+"""
+
+from typing import Any, Optional
+
+import torch
+import torch.nn.functional as F
+from torch import Tensor
+
+
+class ToTensor(object):
+    r"""Convert an array_like data to a tensor of the same shape. This class provides a callable object that allows
```

Comment:
@XianyuanLiu Can you please take a look at this and let me know your opinion? Thanks.

Reply:
[`torchvision.transforms.ToTensor`](https://pytorch.org/vision/stable/generated/torchvision.transforms.ToTensor.html) can convert a PIL Image or ndarray to a callable tensor, but it includes a normalization process ([0, 255] -> [0.0, 1.0]). I am not sure if it is applicable for the data in this example. I have not found other existing functions available for this purpose yet.